backend/controller/admin/admin.go

```diff
@@ -101,8 +109,13 @@ func configProviderKey(p *ftlv1.ConfigProvider) string {
 
 // ConfigSet sets the configuration at the given ref to the provided value.
 func (s *AdminService) ConfigSet(ctx context.Context, req *connect.Request[ftlv1.SetConfigRequest]) (*connect.Response[ftlv1.SetConfigResponse], error) {
+	err := s.validateAgainstSchema(ctx, false, cf.NewRef(*req.Msg.Ref.Module, req.Msg.Ref.Name), req.Msg.Value)
```

Comment:
Module is optional in the proto, so this (and elsewhere) will panic. Use the getter instead.

Reply:
These calls all seem to have been working in prod, but I fixed them up. Good catch!